docs/api/protocol.md

```diff
@@ -1,7 +1,7 @@
 # protocol
 
-The `protocol` module can register a custom protocol or intercept an existing
-protocol.
+> Register a custom protocol to intercept click events from other running
+applications.
```

Comment:
`other running applications` is kind of misleading, `protocol` module can only be used to intercept an existing protocol or register a custom protocol within the application. The previous one-liner seems better for this one.